readthedocs/analytics/proxied_api.py

```diff
@@ -53,11 +53,12 @@ def get(self, request, *args, **kwargs):
         project = self._get_project()
         version = self._get_version()
         absolute_uri = self.request.GET.get('absolute_uri')
-        self.increase_page_view_count(
-            project=project,
-            version=version,
-            absolute_uri=absolute_uri,
-        )
+        if absolute_uri:
```

Comment:
If `absolute_uri` is not present it should return a 400 explaining the issue. Otherwise, it users of this endpoint will think they are using it correctly when it's not.

Reply:
Users shouldn't be using this, this an internal API, but +1 ok returning 400.